dockstore-common/src/main/java/io/dockstore/common/yaml/Author.java

```diff
@@ -0,0 +1,72 @@
+/*
+ *    Copyright 2021 OICR
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package io.dockstore.common.yaml;
+
+import javax.validation.constraints.NotNull;
+
+public class Author {
```

Comment:
You could have constraints for email and/or orcid as well. But if you did, then a .dockstore.yml with an email/orcid would fail. Would we want that? I don't know.